### DISCUSSION TITLE
I am not able to see toggle list with curl

### DISCUSSION THREAD

**Discussion by siddharth-TW** [Unleash/unleash#2508] [Q&A] [answered]
I'm exploring unleash , so while using it's free version , I run the following command
curl http://localhost:3000/proxy -H "Authorization: some-secret"

I am getting empty toggle list , but i have 4 toggles setup (1 from cmd and 3 from UI)

I am using docker to setup server and proxy!

**Comment by sighphyre** [comment, marked as answer]
Hey @siddharth-TW I'm guessing when you say docker, you mean this project here: https://github.com/Unleash/unleash-docker?

What's likely happening is that none of your toggles are enabled, the proxy won't return toggles that evaluate to false. By default the unleash-docker project uses a development token, so would you mind turning one of your toggles on for the development environment with a default strategy and seeing if it gets returned by the proxy?

**Reply by siddharth-TW** [reply]
All of my toggles are on in both the environments

**Reply by siddharth-TW** [reply]
I am using option 1(docker) in this guide to setup https://docs.getunleash.io/reference/deploy/getting-started#option-one---use-docker

**Reply by sighphyre** [reply]
Would you mind sharing the code and proxy keys you're using to setup the proxy?

**Reply by siddharth-TW** [reply]
For server creation:-- 

docker run \
  -p 4242:4242 \
  -e DATABASE_HOST=postgres \
  -e DATABASE_NAME=unleash \
  -e DATABASE_USERNAME=USERNAME\
  -e DATABASE_PASSWORD=PASSWORD\
  -e DATABASE_SSL=false \
  --network unleash \
  --name unleash \
  --pull=always unleashorg/unleash-server


For proxy creation:--

docker run \
  -e UNLEASH_PROXY_CLIENT_KEYS=some-secret \
  -e UNLEASH_URL='http://unleash:4242/api/' \
  -e UNLEASH_API_TOKEN=‘ADMIN-TOKEN-HERE'\
  -p 3000:3000 \
  --network unleash \
  --name unleash-proxy \
  --pull=always unleashorg/unleash-proxy

for checking proxy end point:--
curl http://localhost:3000/proxy -H "Authorization: some-secret"

this returns 
{"toggles":[]}%

**Reply by sighphyre** [reply]
Ah right! So two things here, first, you'll need to use a client token in the proxy for UNLEASH_API_TOKEN, the second thing is that that token needs to exist in Unleash for the proxy to sync. You can do that by setting an environment variable for the proxy:

`-e INIT_CLIENT_API_TOKENS="default:development.unleash-insecure-api-token"`

And another for the server:

`-e UNLEASH_API_TOKEN="default:development.unleash-insecure-api-token"`

Note that the two tokens here need to match exactly.

**Reply by siddharth-TW** [reply]
Thank you @sighphyre  , that helped !

**Reply by sighphyre** [reply]
Awesome! Glad you came right!
